### PR TITLE
LSF attributes

### PR DESCRIPTION
**Pull request by jidur**

### CODE REVIEW

File: lib/npg_seq_melt/merge/generator.pm [hunk 1]
```diff
@@ -167,6 +167,33 @@ has 'tokens_per_job' => ( isa            => 'Int',
...
+=head2 lsf_num_processors
+
+Number of tasks for a parallel job (default 3). Used with the LSF -n option
+
+=cut
+
+has 'lsf_num_processors' => ( isa           => 'Int',
+                              is            => 'ro',
+                              default       => 3,
+                              required      => 0,
```
Comment (dkj) [inline lib/npg_seq_melt/merge/generator.pm]: Does ```required      => 0``` makes sense with a default set?

File: lib/npg_seq_melt/merge/generator.pm [hunk 2]
```diff
@@ -710,7 +737,8 @@ sub _lsf_job_resume {
   my ($self, $job_id) = @_;
   # check child error
   my $LSF_RESOURCES  = q(  -M6000 -R 'select[mem>6000] rusage[mem=6000,seq_merge=) . $self->tokens_per_job()
-                     . q(] span[hosts=1]' -n 3);
+                     . q(] span[hosts=1]' -n ) . $self->lsf_num_processors()
+                     . q( -W ) . $self->lsf_runtime_limit();
```
Comment (dkj) [inline lib/npg_seq_melt/merge/generator.pm]: How about only adding the ```-W``` option if ```$self->lsf_runtime_limit()``` is "true"

**Comment by dkj** [comment]
Do we need to add run_dir to the base.pm module?

**Comment by jidur** [comment]
The update removing the run_dir attribute from 
merge.pm had not made it through : it should only be in generator.pm and 
base.pm.


On Mon, 8 Feb 2016, dkj wrote:

> 
> Do we need to add run_dir to the base.pm module?
> 
> —
> Reply to this email directly or view it on GitHub.[AGp6Hhd44NyQjTSG3sLQ-XElmtoCv8Dkks5piFtTgaJpZM4G-FN2.gif]
> 
> 
>


-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.

**Pull request merged**